cmd/internal/cli/inspect_darwin.go

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package cli
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"sort"
+	"strconv"
+
+	"github.com/spf13/cobra"
+	"github.com/sylabs/sif/pkg/sif"
+	"github.com/sylabs/singularity/docs"
+	"github.com/sylabs/singularity/internal/pkg/sylog"
+)
+
+func init() {
+	cmdManager.RegisterCmd(InspectCmd)
+
+	cmdManager.RegisterFlagForCmd(&inspectDeffileFlag, InspectCmd)
+	cmdManager.RegisterFlagForCmd(&inspectJSONFlag, InspectCmd)
+	cmdManager.RegisterFlagForCmd(&inspectLabelsFlag, InspectCmd)
+}
+
+// InspectCmd represents the 'inspect' command.
+var InspectCmd = &cobra.Command{
+	DisableFlagsInUseLine: true,
+	Args:                  cobra.ExactArgs(1),
+
+	Use:     docs.InspectUse,
+	Short:   docs.InspectShort,
+	Long:    docs.InspectLong,
+	Example: docs.InspectExample,
+
+	Run: func(cmd *cobra.Command, args []string) {
+		f, err := os.Stat(args[0])
+		if os.IsNotExist(err) {
+			sylog.Fatalf("Container not found: %s\n", err)
```

Comment:
```suggestion
			sylog.Fatalf("container not found: %s\n", err)
```

Reply:
Why do you want these `Fatalf` messages un-capitalized? I left them cap to be consistent, because, afaik all the other fatal messages start with a cap. I'm fine either way.

Reply:
Logs usually start with a capital, while errors with a lowercase

Reply:
Okay, for now I will leave the logs starting with a cap.